apis/netacl/v1beta1/zz_acl_types.go

```diff
@@ -75,17 +75,17 @@ type ACLParameters struct {
 
 	// References to VPCSubnet in vpc to populate subnets.
 	// +kubebuilder:validation:Optional
-	SubnetRefs []v1.Reference `json:"subnetRefs,omitempty" tf:"-"`
+	SubnetIDRefs []v1.Reference `json:"subnetIdRefs,omitempty" tf:"-"`
```

Comment:
Be carreful, the principal arguments is `Subnets` and the selector will be `SubnetIDSelector`. Be careful not to disturb the users.